### ISSUE TITLE
Enabling bash program on macOS doesn't install bash

### ISSUE THREAD

**Issue by davidarmstronglewis** [rycee/home-manager#1347]
### Issue description

Unlike other shells I've tried enabling such as `zsh`, `fish`, and `elvish`. When I enable the `bash` program, home-manager does not install the program to my nix profile.

Installing bash using `nix-env --install bash` does make the program available in my nix-profile.

#### Maintainer CC

@rycee
@Gerschtli
@marsam
@maralorn
@jwiegley
@gmarmstrong

#### Technical details

```
 - system: `"x86_64-darwin"`
 - host os: `Darwin 19.5.0, macOS 10.15.5`
 - multi-user?: `no`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.3.6`
 - channels(davidlewis): `"home-manager, nixpkgs-20.09pre230446.a84cbb60f02"`
 - nixpkgs: `/Users/davidlewis/.nix-defexpr/channels/nixpkgs`
```

**Comment by Gerschtli** [comment]
It seems that `bashInteractive` is not installed by this module. Apparently it was assumed to be installed inthe correct version.

You can add `pkgs.bashInteractive` in `home.packages` in `modules/programs/bash.nix` to fix this issue.

**Comment by davidarmstronglewis** [comment]
I'm temted to close this since it seems there are multiple version of bash available (4.4 and 5.0) and it would make sense for this module to not be opinionated about that. Also, I'm not comfortable enough with the Nix expression language just yet to make a contribution.

I'll keep this open for a few more days in case someone knows what the right thing to do is, then I'll close it.
